public/main.css

```diff
@@ -504,6 +509,7 @@ tbody {
   background: #05a700;
   border: 1px solid #05a700;
   cursor: auto;
+  opacity: 0.3;
```

Comment:
Just a question of opinion but for me it's a better visual feedbacks for the users